package.json

```diff
@@ -66,5 +65,6 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"
-  }
+  },
+  "optionalDependencies": {}
```

Comment:
Did you mean to move ember-cli-changelog here? regardless, we really should update it soon :D

Reply:
https://github.com/html-next/ember-hammertime/issues/58 <-- at least we'll track it for tonight